README.md

```diff
@@ -1,23 +1,19 @@
 # :sparkles:Making Data, the DataMade Way:sparkles:
 
-[![Join the chat at https://gitter.im/datamade/data-making-guidelines](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/datamade/data-making-guidelines?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Don't think DataMade wants to commit to be on call for any real-time questions people have about makefiles.